Первый зачет/Зачет 1.1/main.c

```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+
+#define EXPRESSION_SIZE 50
+
+void smallNumber(char num[], size_t size)
+{
+	for (int i = 0; i < size - 1; ++i)
+	{
+		bool swap = false;
+
+		for (int j = 0; j < size - i - 1; ++j)
+		{
+			if (num[j] > num[j + 1])
+			{
+				char temporaryValue = num[j];
+				num[j] = num[j + 1];
+				num[j + 1] = temporaryValue;
+				swap = true;
+			}
+		}
+
+		if (swap == false)
```

Comment:
Стайлгайд запрещает сравниваться с булевыми константами, правильно так:
```suggestion
		if (!swap)
```
Иначе у читающего возникнет мысль, что автор не воспринимает булеву алгебру как алгебру :)